highlighters/main/main-highlighter.zsh

```diff
@@ -1098,6 +1099,20 @@ _zsh_highlight_main_highlighter_highlight_argument()
     base_style=$REPLY
     _zsh_highlight_main_highlighter_highlight_path_separators $base_style
     highlights+=($reply)
+  elif (( $#globs )); then
+    local glob
+    local -a files; files=($~arg(NY1))
+    if ! [[ -e $files[1] ]]; then
```

Comment:
Minor issue, but we could avoi the `stat` by simply checking `$#files` (it'll 0 or 1 so no need for the faster `$+files[1]`).